RecoBTag/TensorFlow/interface/SecondaryVertexConverter.h

```diff
@@ -13,7 +13,8 @@ namespace btagbtvdeep {
   
   void svToFeatures( const reco::VertexCompositePtrCandidate & sv,
 		     const reco::Vertex & pv, const reco::Jet & jet,
-		     SecondaryVertexFeatures & sv_features) ;
+		     SecondaryVertexFeatures & sv_features,
+		     const bool flip_ = false) ;
```

Comment:
please do not use trailing underscore for local variables, this is usually used for member data